src/main/java/appeng/client/gui/implementations/GuiNetworkStatus.java

```diff
@@ -115,7 +116,7 @@ public void drawScreen(final int mouseX, final int mouseY, final float btn) {
     @Override
     public void drawFG(final int offsetX, final int offsetY, final int mouseX, final int mouseY) {
         final ContainerNetworkStatus ns = (ContainerNetworkStatus) this.inventorySlots;
-
+        final DecimalFormat df = new DecimalFormat("#.##");
```

Comment:
Can this be moved into a static private field so it can be reused instead of initialized every frame?
Every little bit helps

Reply:
i'm on it

Reply:
done